plugin/advanced_subscription/src/AdvancedSubscriptionPlugin.php

```diff
@@ -208,20 +208,9 @@ public function isAllowedToDoRequest($userId, $params = array(), $collectErrors
             }
         }
 
-        $checkInduction = $plugin->get('check_induction');
-        // @TODO: check if user have completed at least one induction session
-        $completedInduction = true;
-
-        if ($checkInduction && !$completedInduction) {
-            $this->errorMessages[] = $this->get_lang('AdvancedSubscriptionIncompleteInduction');
-
-            if (!$collectErrors) {
-                throw new \Exception($this->get_lang('AdvancedSubscriptionIncompleteInduction'));
-            }
-        }
-
+        $yearlyCostLimit = $plugin->get('yearly_cost_limit');
         $uitMax = $plugin->get('yearly_cost_unit_converter');
```

Comment:
Please call $uitMax something like $maxCost to avoid being too specific on a language term (uit goes for fiscal tax unit, but it mainly means a cost limit converter here)